website/docs/reference/model-configs.md

```diff
@@ -423,3 +423,30 @@ models:
 ```
 
 </File>
+
+<VersionBlock firstVersion="1.10">
+
+### Configuring source freshness
+
+The model `freshness` config rebuilds models only when new source or upstream data is available. This is useful for models that depend on other models but only need to be updated periodically. For more information, see [freshness](/reference/resource-configs/freshness).
+
+See the following example of a `model.yml` file using the `freshness` config:
```

Comment:
We're pretty flexible on the filename convention used in examples, but just to avoid confusion while scanning, it might be helpful to add a little prefix to the model name to distinguish it more from the directory name
```suggestion
See the following example of a `my_model.yml` file using the `freshness` config:
```